src/hotspot/share/gc/shenandoah/shenandoahGeneration.hpp

```diff
@@ -26,6 +26,7 @@
 #define SHARE_VM_GC_SHENANDOAH_SHENANDOAHGENERATION_HPP
 
 #include "memory/allocation.hpp"
+#include "gc/shenandoah/shenandoahAffiliation.hpp"
```

Comment:
Nit, but `shenandoah/shenandoahAffiliation.hpp` should come between `heuristics/shenandoahSpaceInfo.hpp` and `shenandoah/shenandoahGenerationType.hpp`.

Reply:
Fixed, thanks.